### PR TITLE
Add of Porno Mineiro presence

### PR DESCRIPTION
**Pull request by moitinhors**
This pull request contains the Porno Mineiro presence.

### CODE REVIEW

File: programs/P/PornoMineiro/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "author": {
+    "name": "moitinhu",
+    "id": 714264976067657809
+  },
```
Comment (Bas950) [inline programs/P/PornoMineiro/dist/metadata.json:5]: id should be a string

File: programs/P/PornoMineiro/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "en": "PornoMineiro is a Brazilian porn movies streaming site with the most varied options and categories of porn videos.",
+    "pt_BR": "Porno Mineiro é um site brasileiro de filmes pornos com os mais variados opções e categorias de vídeos pornô."
+  },
+  "url": [
+    "pornomineiro.com",
+    "www.pornomineiro.com"
+  ],
+  "version": "1.0.0",
+  "logo": "https://www.pornomineiro.com/wp-content/uploads/2015/08/favicon.png",
+  "thumbnail": "https://www.pornomineiro.com/wp-content/uploads/2015/08/favicon.png",
```
Comment (Bas950) [inline programs/P/PornoMineiro/dist/metadata.json:17]: upload to imgur

File: programs/P/PornoMineiro/dist/presence.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        presence.setTrayTitle();
+        if (title !== null && views !== null) {
+            presence.setActivity(presenceData);
+        }
+    }
+    else {
+        presence.setActivity();
+        presence.setTrayTitle();
+    }
+});
```
Comment (Bas950) [inline programs/P/PornoMineiro/dist/presence.js:24]: dont upload js file

File: programs/P/PornoMineiro/presence.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
+var presence = new Presence({
+    clientId: '772971071673860127'
+});
+
+presence.on("UpdateData", () => {
+
+  var article: any = document.querySelector(
+    '.clmb08 article'
+  );
```
Comment (Bas950) [inline programs/P/PornoMineiro/presence.ts:9]: no var

File: programs/P/PornoMineiro/presence.ts [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  var article: any = document.querySelector(
+    '.clmb08 article'
+  );
+
+  if(article) {
+
+    let title: any;
+    title = (document.querySelector('.ttl-cn.dfl.fa-film h2') as any).innerText;
+
+    let views: any;
```
Comment (Bas950) [inline programs/P/PornoMineiro/presence.ts:16]: no any

**Review by Bas950 — CHANGES_REQUESTED** [review]
This is inside of the wrong folder... it should be in the websites folder and not programs

**Comment by ririxidev** [comment]
Why you even did that in programs folder? huh? We do not support programs yet.

**Pull request closed without merging**